src/challenges.js

```diff
@@ -28,16 +52,38 @@ const cacheFunction = cb => {
 
 /* ======================== Recursion Practice ============================ */
 const reverseStr = str => {
+  return str.split('').reverse().join('');
   // reverse str takes in a string and returns that string in reversed order
   // The only difference between the way you've solved this before and now is that you need to do it recursivley!
 };
 
 const checkMatchingLeaves = obj => {
```

Comment:
Great solution.  Doesn't check for nested objects (this is when recursion would have come in) but the solution for the first test is very nice.